### DISCUSSION TITLE
Is there a possibility to generate several batches at a time?

### DISCUSSION THREAD

**Discussion by Renaldas111** [Comfy-Org/ComfyUI#870] [General]
I cant find anywhere in ComfyUI, how to set batch size to >1 during inpainting or img2img. Is this option somewhere hidden?

**Comment by sccook5** [comment]
I expect you found a solution to this long ago, but for anyone else with the same issue, here's a solution.

In the Node Library, search for a node called Repeat Latent Batch, and insert it into the workflow, somewhere between the node where the latent gets created and the Sampler node (very likely, there will only be one place where it can fit).
